tests/e2e/taqueria-plugin-taquito.spec.ts

```diff
@@ -0,0 +1,346 @@
+            const stdoutDeploy = execSync(`taq deploy -e ${environment}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 3. Verify that proper error displays in the console
+            expect(stdoutDeploy).toContain("E_INVALID_PLUGIN_RESPONSE");
+            expect(stdoutDeploy).toContain('\\"status\\":\\"failed\\"');
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+    // TODO: Currently there is no specific message for this type of error
+    // Issue associated https://github.com/ecadlabs/taqueria/issues/313
+    test.skip('Verify that taqueria taquito plugin will show proper error when configuration is wrong -> empty', async () => {
+        try {
+            // Environment test does not exist on default config.json
+            environment = "test"
+
+            // 1. Copy config.json and two michelson contracts from data folder to artifacts folder under taqueria project
+            execSync(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/artifacts/hello-tacos.tz`);
+            execSync(`cp e2e/data/config-taquito-test-environment-invalid-faucet-empty.json ${taqueriaProjectPath}/.taq/config.json`);
+
+            // 2. Run taq deploy on a network described in "test" environment
+            const stdoutDeploy = execSync(`taq deploy -e ${environment}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 3. Verify that proper error displays in the console
+            expect(stdoutDeploy).toContain("E_INVALID_PLUGIN_RESPONSE");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+    // TODO: Currently there is no specific message for this type of error
+    // Issue associated https://github.com/ecadlabs/taqueria/issues/313
+    test.skip('Verify that taqueria taquito plugin will show proper error when configuration is wrong -> invalid pkh', async () => {
+        try {
+            // Environment test does not exist on default config.json
+            environment = "test"
+
+            // 1. Copy config.json and two michelson contracts from data folder to artifacts folder under taqueria project
+            execSync(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/artifacts/hello-tacos.tz`);
+            execSync(`cp e2e/data/config-taquito-test-environment-invalid-faucet-pkh.json ${taqueriaProjectPath}/.taq/config.json`);
+
+            // 2. Run taq deploy on a network described in "test" environment
+            const stdoutDeploy = execSync(`taq deploy -e ${environment}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 3. Verify that proper error displays in the console
+            expect(stdoutDeploy).toContain("E_INVALID_PLUGIN_RESPONSE");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+    // TODO: Currently there is no specific message for this type of error
```

Comment:
Please see comment above. We can probably create one issue for enabling any of these skipped tests that are dependent on this problem being resolved.

Reply:
Fixed. Issue to investigate and re-enable tests - https://github.com/ecadlabs/taqueria/issues/377